Client/Ecosia/Migration/LoadingScreen.swift

```diff
@@ -8,14 +8,16 @@ final class LoadingScreen: UIViewController {
     private weak var profile: Profile!
     private weak var tabManager: TabManager!
     private weak var progress: UIProgressView!
+    private let referrals: Referrals
```

Comment:
Let's make them `weak` to avoid unnecessarily increasing the reference count and retain cycles(maybe not an issue but still), but also to make it explicit that this is not the owner.

Reply:
I think both ways will work and I am sure there are no retain cycles at the moment, and even if the referrals instance won't be deallocated anyhow with the current setup as BrowserViewController (which holds it) outlives the whole time. Still let's do things right 👍 

What I like about (strong) let is, that it forces you to assign or init it during init of the class itself and is therefore not forgotten to be assigned. So I prefer it slightly over weak var forced unwrapped, but that's personal taste. So happy to change it here :)

Reply:
Good points that you express and very valid 👍 . As an alternative to avoid forgetting to assign it a test could help.